crypto/fipsmodule/rand/rand.c

```diff
@@ -392,6 +392,8 @@ void RAND_bytes_with_additional_data(uint8_t *out, size_t out_len,
       CRYPTO_STATIC_MUTEX_unlock_write(thread_states_list_lock_bss_get());
     }
 #endif
+    OPENSSL_cleanse(seed, CTR_DRBG_ENTROPY_LEN);
+    OPENSSL_cleanse(personalization, CTR_DRBG_ENTROPY_LEN);
```

Comment:
Are we concerned at all about the performance implications of these additional operations? How much will this impact the latency of functions calling `RAND_bytes`?

Reply:
No issue, these are all (except one) during seeding.